dspace/config/item-submission.xml

```diff
@@ -18,6 +18,7 @@
     <!-- for handle "default". -->
     <submission-map>
         <name-map collection-handle="default" submission-name="traditional"/>
+        <name-map collection-handle="123456789/1161" submission-name="People" />
```

Comment:
This shouln't be versioned.

Reply:
Indeed, that was a remnant from a test case that we used. I'll remove this config